### PR TITLE
3 browsing items by category

### PR DESCRIPTION
**Pull request by Aram-Anderson**
# Issue Number: 3

# Description of changes: Liam's work on issue #3 

# @mentions of the person or team responsible for reviewing proposed changes:
@dphilla @mikeyduece

### CODE REVIEW

**Review by Aram-Anderson — COMMENTED** [review]
Liam's branch. Looks good.

File: app/controllers/tags_controller.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class TagsController < ApplicationController
+
+  def index
+
+  end
+
+  def show
+    @tag = Tag.find_by(name: params[:name])
```
Comment (dphilla) [inline app/controllers/tags_controller.rb:8]: This is solid way to get that custom wildcard

File: spec/features/user_can_see_items_by_category_spec.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.feature "user can see items by category" do
+  scenario "from main page" do
+    tag1, tag2 = create_list(:tag, 2)
+    item1, item2, item3 = create_list(:item, 3)
+    item_tag1 = create(:item_tag, tag: tag1, item: item1)
```
Comment (dphilla) [inline spec/features/user_can_see_items_by_category_spec.rb:7]: Is this necessary to do?

File: config/routes.rb [hunk 3]
```diff
@@ -2,6 +2,7 @@
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
 
  resources :items
+ resources :tags, only: [:index, :show]
 
+ get "/:name" => "tags#show"
```
Comment (dphilla) [inline config/routes.rb:7]: How will this affect path helpers?

**Review by dphilla — COMMENTED** [review]
nice work

File: spec/factories/item_tags.rb [hunk 4]
```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
```
Comment (Aram-Anderson) [inline spec/factories/item_tags.rb:1]: I don't think we need this. The joins table will automatically populate with the relationship in place.

**Review by mikeheft — APPROVED** [review]
Looks ok to me

**Review by dphilla — COMMENTED** [review]
looks good!

File: spec/features/user_can_see_items_by_category_spec.rb [hunk 5]
```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.feature "user can see items by category" do
+  scenario "from main page" do
```
Comment (Aram-Anderson) [inline spec/features/user_can_see_items_by_category_spec.rb:4]: Again, not sure we need to create item_tags. We just need Item.tags.create.

**Pull request merged**